src/Application/Engine.cpp

```
@@ -55,6 +41,18 @@ void Engine::init() {
 }
 
 
+void Engine::loadMainWindow() {
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)